### PR TITLE
Changes for Cypress 10.0.0

### PR DESCRIPTION
**Pull request by flotwig**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   },
   "devDependencies": {
     "axe-core": "3.5.5",
-    "cypress": "9.3.1",
+    "cypress": "https://cdn.cypress.io/beta/npm/10.0.0/linux-x64/10.0-release-b9af1054a860aea35ec0d6e935174f5a50a0bbf6/cypress.tgz",
```
Comment (admah) [inline package.json:36]:
```suggestion
    "cypress": "10.0.0",
```

**Review by admah — CHANGES_REQUESTED** [review]
Looks good. I just suggested updating the Cypress version in package.json.

File: cypress.config.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
+  fixturesFolder: false,
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
```
Comment (mjhenkes) [inline cypress.config.ts:7]: nit, you could remove the comment

Comment (flotwig) [reply cypress.config.ts]: 85e2880

**Pull request merged**